Worktez/worktez
# fix(Milestone): Fixed Indexing for Milestone

### Functionality:
Fixed Indexing for Tasks Collection
Fixed Milestones Removing on Edited

### Solution:
Added new Indexing in Tasks collection for Milestones

### Risk level:

### How to test:
Check whether the Add task in Milestone is working
Check whether the task remains on the milestone on edit
